packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1161,10 +1161,21 @@ export class WindowedList<
       this.viewModel.scrollOffset = scrollOffset;
       this._scrollUpdateWasRequested = false;
 
+      if (this._viewport.dataset.isScrolling != 'true') {
+        this._viewport.dataset.isScrolling = 'true';
+      }
+
+      if (this._timerToClearScollStatus) {
+        window.clearTimeout(this._timerToClearScollStatus);
+      }
+      this._timerToClearScollStatus = window.setTimeout(() => {
+        this._viewport.dataset.isScrolling = 'false';
```

Comment:
[`scrollend`](https://developer.mozilla.org/en-US/docs/Web/API/Document/scrollend_event) event is a thing but not in Safari (https://bugs.webkit.org/show_bug.cgi?id=201556)

Reply:
We could add a comment about that, to fix it when it is available in Safari.